selectors-4/Overview.bs

```diff
@@ -2280,19 +2269,34 @@ The Focus-Indicated Pseudo-class: '':focus-visible''</h3>
 	but with different user settings,
 	can make different choices as to when an element matches '':focus-visible''.
 
-	<div class=example>
-		The following guidelines are suggested heuristics
-		for choosing when to apply '':focus-visible''
-		to elements without “native” focus indicator behavior:
-
-		* If the element received focus via a keyboard interaction,
-			including indirectly,
-			such as triggering a dialog
-			by pressing a button using the keyboard,
-			apply '':focus-visible''.
-		* If a keyboard event occurs while an element is focused,
-			even if the element wasn't focused by a keyboard interaction,
-			apply '':focus-visible''.
+	Since part of the problem is user agents' default focus indicator behaviour,
```

Comment:
I wonder if "the problem" should be spelled out in slightly more words, as well, since it doesn't seem tied to the previous paragraph.